src/azul/plugins/repository/tdr_anvil/__init__.py

```diff
@@ -113,7 +115,97 @@ def merge(cls, links: Iterable['Link']) -> 'Link':
 Links = set[Link]
 
 
-class Plugin(TDRPlugin):
+class BundleEntityType(Enum):
+    primary: EntityType = 'biosample'
+    supplementary: EntityType = 'file'
+
+
+@attr.s(auto_attribs=True, frozen=True, kw_only=True)
+class AnvilBundleFQID(TDRBundleFQID):
+    entity_type: BundleEntityType
+
+    def fqid_json(self) -> MutableJSON:
+        result = super().fqid_json()
```

Comment:
```suggestion
    def as_json(self) -> MutableJSON:
        result = super().as_json()
```

Reply:
Decided at PL to change shape of the bundle notification so that `source` can be included in the return value of this method.